src/main/java/com/hbm/blocks/ModBlocks.java

```diff
@@ -1712,8 +1709,7 @@ private static void initializeBlock() {
 		
 		machine_puf6_tank = new MachinePuF6Tank(Material.iron).setBlockName("machine_puf6_tank").setHardness(5.0F).setResistance(10.0F).setCreativeTab(MainRegistry.machineTab);
 		
-		machine_reactor = new MachineReactor(Material.iron).setBlockName("machine_reactor").setHardness(5.0F).setResistance(10.0F).setCreativeTab(MainRegistry.machineTab).setBlockTextureName(RefStrings.MODID + ":machine_reactor");
-		machine_reactor_on = new MachineReactor(Material.iron).setBlockName("machine_reactor_on").setHardness(5.0F).setResistance(10.0F).setCreativeTab(null).setBlockTextureName(RefStrings.MODID + ":machine_reactor_on");
+		machine_reactor_breeding = new MachineReactorBreeding(Material.iron).setBlockName("machine_reactor_breeding").setHardness(5.0F).setResistance(10.0F).setCreativeTab(MainRegistry.machineTab).setBlockTextureName(RefStrings.MODID + ":machine_reactor");

```

Comment:
![image](https://user-images.githubusercontent.com/16674258/148278832-9656d67a-be07-4385-be57-f737c41374a7.png)
Hold on there buster, you're about to pulverize everyone's breeding reactors

Reply:
We already discussed this
if it's too much of an issue you can just change it back once it's merged

Reply:
![ran fumo spin](https://user-images.githubusercontent.com/63076393/148285761-bd1db1e3-c936-4063-be50-a3d483dc4de9.gif)

Reply:
scheiße